source/cli-modules-v1.json

```
@@ -12,6 +12,10 @@
         "PSModuleName": "DockerCompletion",
         "MinimumPowerShellVersion": "5.0"
     },
+    "gcloudsdk": {
```

Shouldn't this be just `gcloud`? posh-cli checks for the installed CLIs and only bootstraps the powershell modules for the CLIs that are available on a given machine.
```suggestion
    "gcloud": {
```

---

Ahh didn't realise this was a reference to the executable name. I thought it was just a generic name...makes a lot more sense 🙂